async/src/main/java/org/odk/collect/async/TaskSpec.kt

```diff
@@ -18,10 +18,4 @@ interface TaskSpec {
      * once instead of doing that after every single execution.
      */
     fun getTask(context: Context, inputData: Map<String, String>, isLastUniqueExecution: Boolean): Supplier<Boolean>
-
-    /**
-     * Returns class that can be used to schedule this task using Android's
-     * WorkManager framework
-     */
-    fun getWorkManagerAdapter(): Class<out WorkerAdapter>
```

Comment:
I realised we didn't need all this and could just use strings (and no arg constructors) to integrate with WorkManager. It will eventually have to store our class names as strings anyway.